packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -1650,6 +1728,47 @@ export const accountRoutes = (
       },
       handler: (request: AuthRequest) => accountHandler.finishSetup(request),
     },
+    {
+      method: 'POST',
+      path: '/account/set_password',
+      options: {
+        ...ACCOUNT_DOCS.ACCOUNT_SET_PASSWORD_POST,
+        auth: {
+          mode: 'required',
+          payload: false,
+          strategy: 'oauthToken',
```

Comment:
Maybe it will be made obvious in a later PR, but I don't understand where this access token was obtained in the first place.

Reply:
`/account/stub` is called first and returns the access token. This sets the password on a stub account.

Reply:
Ah, gotcha.  Thanks for responding to my drive-by comments.